src/plugins/intel_cpu/tests/functional/subgraph_tests/src/fuse_muladd_ewsimple.cpp

```diff
@@ -31,83 +34,88 @@ void FuseMulAddAndEwSimpleTest::SetUp() {
     CreateGraph();
 }
 
-const auto mulAddAndEwSimpleCommonParams = ::testing::Combine(
-        ::testing::Values(SizeVector{1, 20}),
-        ::testing::Values(Precision::FP32)
-);
-
+const auto mulAddAndEwSimpleCommonParams =
+    ::testing::Combine(::testing::Values(ov::Shape({1, 20})), ::testing::Values(ov::element::f32));
 
 // Fused EltwiseAndSimple comes on the 3rd port into MulAdd
 void FuseMulAddAndEwSimpleTest1::CreateGraph() {
-    auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(inPrec);
+    auto ngPrc = inPrec;
     auto mulSecondInput = inputShape;
     mulSecondInput[0] = 1;
-    ov::ParameterVector params{std::make_shared<ov::op::v0::Parameter>(ngPrc, ov::Shape(inputShape)),
-                               std::make_shared<ov::op::v0::Parameter>(ngPrc, ov::Shape(inputShape)),
-                               std::make_shared<ov::op::v0::Parameter>(ngPrc, ov::Shape(mulSecondInput))};
+    ov::ParameterVector params{std::make_shared<ov::op::v0::Parameter>(ngPrc, inputShape),
+                               std::make_shared<ov::op::v0::Parameter>(ngPrc, inputShape),
+                               std::make_shared<ov::op::v0::Parameter>(ngPrc, mulSecondInput)};
 
-    auto clamp = ngraph::builder::makeActivation(params[0], ngPrc, ActivationTypes::Clamp, inputShape, {0, 100});
-    auto tanh = ngraph::builder::makeActivation(clamp, ngPrc, ActivationTypes::Tanh);
-    auto mul1 = ngraph::builder::makeEltwise(params[1], params[2], EltwiseTypes::MULTIPLY);
-    auto add = ngraph::builder::makeEltwise(tanh, mul1, EltwiseTypes::ADD);
+    auto clamp = ov::test::utils::make_activation(params[0], ngPrc, ActivationTypes::Clamp, inputShape, {0, 100});
+    auto tanh = ov::test::utils::make_activation(clamp, ngPrc, ActivationTypes::Tanh);
+    auto mul1 = ov::test::utils::makeEltwise(params[1], params[2], EltwiseTypes::MULTIPLY);
```

Comment:
@olpipi may we rename builder to `make_eltwise`?

Reply:
Oh.. This is my mistake. I renamed other builders to snake_case, but missed this one.
I will rename it soon